arwen/contexts/runtime.go

```diff
@@ -643,16 +752,9 @@ func (context *runtimeContext) GetInstanceExports() wasmer.ExportsMap {
 	return context.instance.GetExports()
 }
 
-// CleanWasmerInstance cleans the current wasmer instance.
+// CleanWasmerInstance cleans the current wasmer instance. it does nothing as cache and eviction are cleaning
```

Comment:
Please remove this function.
We have `CleanWasmerInstance` and `ClearWarmInstanceCache`. Or make the CleanWasmerInstance contain the ClearWasmerInstance code. 
Otherwise this will confuse reviewers and maintainers of this code (felt in the the pitfall, thinking that you just used a useless function just about anywhere).

Reply:
done